src/embed/scene-info.js

```diff
@@ -52,7 +52,8 @@ SceneInfo.loadFromGetParams = function() {
   var params = {};
   for (var camelCase in CAMEL_TO_UNDERSCORE) {
     var underscore = CAMEL_TO_UNDERSCORE[camelCase];
-    params[camelCase] = Util.getQueryParameter(underscore);
+    params[camelCase] = Util.getQueryParameter(underscore)
+                        || ((window.WebVRConfig && window.WebVRConfig.PLAYER) ? window.WebVRConfig.PLAYER[underscore] : undefined);
```

Comment:
this should be "", not undefined, to be consistent with the old code (Util.getQueryParameter returns "" if the parameter is not found).